### DISCUSSION TITLE
The real time line between the approvement and getting the benefits !

### DISCUSSION THREAD

**Discussion by HoussamEssaki** [community/community#84881] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,
Hello everyone,

![IMG_20240107_231858](https://github.com/community/community/assets/116951226/e2c348ae-a477-4c8b-adee-82fd369ad2c3)

I have recently applied for the student developer pack. I still have not received my benefits even though it says it was approved. It usually takes 72 hours to get the benefits but it has already been 3 days for me as it got approved on January 4th. I just wanna know if something is wrong with my application and how long I have to wait before getting the benefits.
I'll be really thankful for any help. Much appreciated.
Thanks for taking your time and reading this.

### A suggestion...(optional)

Responding on college students questions concernant getting the student developer pack.

**Comment by HoussamEssaki** [comment]
@samus-aran Thanks for reviewing the following request.

**Reply by HoussamEssaki** [reply, marked as answer]
Thankkies . A Real active community 💪
![IMG_20240107_234244](https://github.com/community/community/assets/116951226/117f85c9-5dc1-48f8-98b0-a2094a800b1a)

**Reply by bello-kirito** [reply]
@samus-aran @HoussamEssaki 
I've also got my request approved today, should i wait for 3 days to get the benefits as well ?
![Screenshot_2024-01-09-13-10-25-05_e4424258c8b8649f6e67d283a50a2cbc.jpg](https://github.com/community/community/assets/153209833/31563b2f-682c-4812-8f19-075c1e8db900)

Thank u

**Reply by HoussamEssaki** [reply]
@bello-kirito Yes absolutely. At least you should wait up to 72 hours ( 3 days) , and take notice that the process may take longer than that in some cases. Your benefits will be available as soon as possible.
Have a great day.

**Reply by bello-kirito** [reply]
@HoussamEssaki  Okay, i understand now, Thanks to you !
Good day 😊☀️
